ingenialink/exceptions.py

```diff
@@ -1,74 +1,50 @@
 class ILError(Exception):
     """IngeniaLink generic error."""
 
-    pass
-
 
 class ILConfigurationError(ILError):
     """IngeniaLink configuration error."""
 
-    pass
-
 
-class ILUDPException(ILError):
+class ILUDPExceptionError(ILError):
```

Comment:
Do we use it somewhere?

Reply:
Yes, on the [UDP ](https://github.com/ingeniamc/ingenialink-python/blob/7.4.0/ingenialink/utils/udp.py#L88)module.

Reply:
And it should always be caught by this except, right?  https://github.com/ingeniamc/ingenialink-python/blob/662718e54dd266f43222a56324e55c126c16c6a2/ingenialink/ethernet/network.py#L181
I don't see any way for the user to catch this exception. If this exception could be seen by a user, the rename could be dangerous.

Reply:
I think that the renaming is safe. The ILUDPExceptionError exception is not seen by the users as the exception is re-raised as a [ILFirmwareLoadError](https://github.com/ingeniamc/ingenialink-python/blob/662718e54dd266f43222a56324e55c126c16c6a2/ingenialink/ethernet/network.py#L183) exception (which was not renamed).